src/Content/Embed/EmbedVideo.tsx

```diff
@@ -37,11 +37,13 @@ function ThumbnailWrapper({
       onClick={() => setHideThumbnail(true)}
       onError={() => setError(true)}
       style={{
-        width: adjustedWidth,
-        height: adjustedHeight,
-        backgroundImage: `url(${error ? discordImageFailure : thumbnail})`,
+        width: "100%",
+        height: "auto",
+        maxWidth: adjustedWidth,
+        maxHeight: adjustedHeight,
       }}
     >
+      <Styles.Image src={error ? discordImageFailure : thumbnail} />
```

Comment:
What does this do?

Reply:
This helps to make the image to be fix on the small container without any cutting at the edges.